File: Node/Sequelize_ORM/src/dal/entities/TestData.js
const Sequelize = require('sequelize');
module.exports = function(sequelize, DataTypes) {
  return sequelize.define('TestData', {
    Id: {
      autoIncrement: true,
      type: DataTypes.INTEGER,
      allowNull: false,
      primaryKey: true
    },
    TestName: {
      type: DataTypes.STRING(50),
      allowNull: true
    },
    CpuUsage: {
      type: DataTypes.STRING(50),
      allowNull: true
    },
    MemoryUsage: {
      type: DataTypes.STRING(50),
      allowNull: true
    },
    Performance: {
      type: DataTypes.STRING(50),
      allowNull: true
    },
    Language: {
      type: DataTypes.STRING(50),
      allowNull: true
    }
  }, {
    sequelize,
    tableName: 'TestData',
    schema: 'dbo',
    timestamps: false,
    indexes: [
      {
        name: "PK__TestData__8CC3310000C56314",
        unique: true,
        fields: [
          { name: "Id" },
        ]
      },
    ]
  });
};

File: Go/SQLBoiler_ORM/dal/entities/testData.go
package entities

type TestData struct {
	ID          int     `db:"Id"`
	TestName    *string `db:"TestName"`
	CpuUsage    *string `db:"CpuUsage"`
	MemoryUsage *string `db:"MemoryUsage"`
	Performance *string `db:"Performance"`
	Language    *string `db:"Language"`
}

func (TestData) TableName() string {
	return "TestData"
}
